create-djs-app.js.org

NotTCA opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

https://create-djs-app.vercel.app

indus commented:
@MattIPv4  I'm having a hard time seeing a relation or comparing it to https://github.com/js-org/js.org/pull/7694
This sounds like a scaffolding tool, that if well made shouldn't need a lot of documentation. I can see how this may be relvant for a Js developer. So I think it ok.

@NotTCA I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain.
I'll merge after we have "contact". Please keep this PR open.

NotTCA commented:
> I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain.

Thanks! I've added the domain on Vercel.

indus commented:
Welcome https://create-djs-app.js.org
